diagnostic.md

```diff
@@ -32,7 +36,13 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+Getters:
+  .find()
+  .val()
 
+Setters:
+  .text()
+  .css()
```

Comment:
these are also getters. A setter would be `.text('value of text you want to set')`